src/ProgramBlockEditor.scss

```diff
@@ -134,6 +134,7 @@ button.btn.ProgramBlockEditor__program-deleteAll-button--pressed.ProgramBlockEdi
     display: flex;
     justify-content: center;
     padding: 0 0.5rem;
+    padding-top: 2.9rem;
```

Comment:
Program sequence has padding-bottom: 1rem set to it, and would be great if we can just set same amount of bottom padding to this character column. We can make character icon to start at the bottom of the column, and give 1rem margin at the bottom of it by setting align-items: self-end. Also, let's remove padding rules in character-column and use padding-bottom: 1rem.

```
.ProgramBlockEditor__character-column {
    align-items: center;
    background-color: #30444E;
    border-right: 1px solid #19282F;
    border-radius: 3px 0 0 3px;
    box-shadow: 2px 0px 1px #8F8F8F;
    display: flex;
    justify-content: center;
    align-items: self-end;
    padding-bottom: 1rem;
```